app/models/person.rb

```diff
@@ -28,11 +28,19 @@ class Person < ActiveRecord::Base
   validates :email, presence: true, unless: :phone
   validates :phone, presence: true, unless: :email
 
+  attr_accessor :address, :zip, :tags
```

Comment:
I'm not sure if it's great to add this `tags` attribute, but it works for now. I was thinking if we end up passing a lot of data through to NB, instead of creating an attribute for each, we could have a single attribute called `:remote_fields`, or something that could be passed straight through